src/EditorFeatures/Core/InlineRename/AbstractEditorInlineRenameService.InlineRenameLocationSet.cs

```diff
@@ -39,7 +39,7 @@ private InlineRenameLocation ConvertLocation(RenameLocation location)
             public async Task<IInlineRenameReplacementInfo> GetReplacementsAsync(string replacementText, SymbolRenameOptions options, CancellationToken cancellationToken)
             {
                 var conflicts = await _renameLocationSet.ResolveConflictsAsync(
-                    _renameInfo.GetFinalSymbolName(replacementText), nonConflictSymbols: null, cancellationToken).ConfigureAwait(false);
+                    _renameInfo.RenameSymbol, _renameInfo.GetFinalSymbolName(replacementText), nonConflictSymbolKeys: default, cancellationToken).ConfigureAwait(false);
```

Comment:
Major points:  

1. LightweightRenameLocations doesn't hold onto the symbol anymore.  So it needs ot be passed along.
2. nonConflictSymbols became SymbolKeys